packages/plugin-credentials/src/credentials.js

```diff
@@ -65,6 +216,49 @@ const Credentials = SparkPlugin.extend({
         const ErrorConstructor = grantErrors.select(res.body.error);
         return Promise.reject(new ErrorConstructor(res._res || res));
       });
+  },
+
+  set(attrs, options) {
```

Comment:
(unless you're using ampersand-model, the parse function is pretty much useless)